_build_cfg.yml

```diff
@@ -88,6 +88,8 @@ Topics:
     Distros: openshift-online
   - Name: Web Console Walkthrough
     File: developers_console
+  - Name: Command-Line Walkthrough
+    File: developers_cli
     Distros: openshift-enterprise,openshift-dedicated,openshift-origin
```

Comment:
Can you make sure both `developers_console` and `developers_cli` get the `openshift-enterprise,openshift-dedicated,openshift-origin` distro? We still have the split between the Online GS using `basic_walkthrough` + `beyond_the_basics`, and the other distros using `developers_console` + `developers_cli`.